scripts/dev_setup.sh

```diff
@@ -564,6 +571,7 @@ if [[ "$INSTALL_BUILD_TOOLS" == "true" ]]; then
   rustup component add rustfmt
   rustup component add clippy
 
+  install_old_sccache
```

Comment:
`install_sccache`? I don't see `install_old_sccache` defined in here.

Reply:
good catch